<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.269">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<title>DAPR2 - Write Up Example &amp; Block 4 Recap</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light"><script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script><script>
function toggle_visibility(id1, id2) {
var e = document.getElementById(id1);
var f = document.getElementById(id2);
e.style.display = ((e.style.display!='none') ? 'none' : 'block');
if(f.classList.contains('fa-chevron-circle-right')) {
    f.classList.add('fa-chevron-circle-down')
    f.classList.remove('fa-chevron-circle-right')
} else {
    f.classList.add('fa-chevron-circle-right')
    f.classList.remove('fa-chevron-circle-down')
}
}
</script><script src="https://kit.fontawesome.com/120b08a6f5.js" crossorigin="anonymous"></script><script src="site_libs/kePrint-0.0.1/kePrint.js"></script><link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>
</head>
<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top"><nav class="navbar navbar-expand-lg navbar-dark "><div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">DAPR2</span>
    </a>
  </div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
<li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-intro-to-linear-models" role="button" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Intro to Linear Models</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-intro-to-linear-models">
<li>
    <a class="dropdown-item" href="./1_01_function.html">
 <span class="dropdown-text">1/1: Functions and Models</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_02_slr.html">
 <span class="dropdown-text">1/2: Intro to Linear Regression</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_03_mlr.html">
 <span class="dropdown-text">1/3: Multiple Regression</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_04_model_fit.html">
 <span class="dropdown-text">1/4: Model Fit and Standardization</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_05_cat_recap.html">
 <span class="dropdown-text">1/5: Cat Predictors &amp; Recap</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-more-linear-models" role="button" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">More Linear Models</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-more-linear-models">
<li>
    <a class="dropdown-item" href="./1_07_int1_nc.html">
 <span class="dropdown-text">1/7: Interactions I: Num x Cat</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_08_int2_nn.html">
 <span class="dropdown-text">1/8: Interactions II: Num x Num</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_09_int3_cc.html">
 <span class="dropdown-text">1/9: Interactions III: Cat x Cat</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_10_assump_diag.html">
 <span class="dropdown-text">1/10: Assumptions &amp; Diagnostics</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./1_11_writeup_recap.html">
 <span class="dropdown-text">1/11: Write Up &amp; Recap</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-analysing-experimental-studies" role="button" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Analysing Experimental Studies</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-analysing-experimental-studies">
<li>
    <a class="dropdown-item" href="./2_01_model_comps.html">
 <span class="dropdown-text">2/1: Model Comparison</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./2_02_effects.html">
 <span class="dropdown-text">2/2: Effects Coding</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./2_03_contrasts.html">
 <span class="dropdown-text">2/3: Contrasts</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./2_04_emmeans.html">
 <span class="dropdown-text">2/4: Simple Effects &amp; Pairwise Comparisons</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./2_05_writeup_recap.html">
 <span class="dropdown-text">2/5: Write Up &amp; Recap</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-advanced-topics-for-lm" role="button" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Advanced Topics for LM</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-advanced-topics-for-lm">
<li>
    <a class="dropdown-item" href="./2_06_glm1.html">
 <span class="dropdown-text">2/6: Binary Logistic Regression</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./2_07_glm2.html">
 <span class="dropdown-text">2/7: More Logistic Regression</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="./2_08_power.html">
 <span class="dropdown-text">2/8: Sample Size &amp; Power Analysis</span></a>
  </li>  
        <li class="dropdown-header">2/9: Write Up &amp; Recap</li>
        <li class="dropdown-header">2/10: Mock Exam</li>
    </ul>
</li>
</ul>
<div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav></header><!-- content --><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active"><h2 id="toc-title">On this page</h2>
   
  <ul>
<li>
<a href="#section-a-write-up" id="toc-section-a-write-up" class="nav-link active" data-scroll-target="#section-a-write-up">Section A: Write-Up</a>
  <ul class="collapse">
<li>
<a href="#study-overview" id="toc-study-overview" class="nav-link" data-scroll-target="#study-overview">Study Overview</a>
  <ul class="collapse">
<li><a href="#setup" id="toc-setup" class="nav-link" data-scroll-target="#setup">Setup</a></li>
  <li><a href="#analysis-code" id="toc-analysis-code" class="nav-link" data-scroll-target="#analysis-code">Analysis Code</a></li>
  <li><a href="#the-3-act-structure-analysis-strategy-results-discussion" id="toc-the-3-act-structure-analysis-strategy-results-discussion" class="nav-link" data-scroll-target="#the-3-act-structure-analysis-strategy-results-discussion">The 3-Act Structure: Analysis Strategy, Results, &amp; Discussion</a></li>
  </ul>
</li>
  </ul>
</li>
  <li><a href="#section-b-weeks-1-5-recap" id="toc-section-b-weeks-1-5-recap" class="nav-link" data-scroll-target="#section-b-weeks-1-5-recap">Section B: Weeks 1-5 Recap</a></li>
  </ul></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block default"><div class="quarto-title">
<h1 class="title">Write Up Example &amp; Block 4 Recap</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header><div class="lo">
<section id="learning-objectives" class="level3"><h3 class="anchored" data-anchor-id="learning-objectives">
<i class="fa fa-graduation-cap"></i> Learning Objectives</h3>
<p>At the end of this lab, you will:</p>
<ol type="1">
<li>Understand how to write-up and provide interpretation of a binary logistic regression model</li>
</ol></section><section id="what-you-need" class="level3"><h3 class="anchored" data-anchor-id="what-you-need">
<i class="fa fa-check-square-o fa-2"></i> What You Need</h3>
<ol type="1">
<li>Be up to date with lectures</li>
<li>Have completed Labs 7-9</li>
</ol></section><section id="required-r-packages" class="level3"><h3 class="anchored" data-anchor-id="required-r-packages">
<i class="fab fa-r-project"></i> Required R Packages</h3>
<p>Remember to load all packages within a code chunk at the start of your RMarkdown file using <code><a href="https://rdrr.io/r/base/library.html">library()</a></code>. If you do not have a package and need to install, do so within the console using <code>install.packages(" ")</code>. For further guidance on installing/updating packages, see Section C <a href="https://uoepsy.github.io/files/install-update-r#update-pkgs">here</a>.</p>
<p>For this lab, you will need to load the following package(s):</p>
<ul>
<li><strong>tidyverse</strong></li>
<li><strong>psych</strong></li>
<li><strong>kableExtra</strong></li>
<li><strong>sjPlot</strong></li>
</ul></section><section id="lab-data" class="level3"><h3 class="anchored" data-anchor-id="lab-data">
<i class="fa fa-file"></i> Lab Data</h3>
<p>You can download the data required for this lab <a href="https://uoepsy.github.io/data/SenilityWAIS.csv">here</a> or read it in via this link https://uoepsy.github.io/data/SenilityWAIS.csv.</p>
</section>
</div>
<section id="section-a-write-up" class="level1"><h1>Section A: Write-Up</h1>
<p>In this section of the lab you will be be presented with a research question, and tasked with writing up and presenting your analyses.</p>
<p>The aim in writing should be that a reader is able to more or less replicate your analyses <strong>without</strong> referring to your <code>R</code> code. This requires detailing all of the steps you took in conducting the analysis. The point of using RMarkdown is that you can pull your results <strong>directly</strong> from the code. If your analysis changes, so does your report!</p>
<p>Make sure that your final report doesn’t show any R functions or code. Remember you are interpreting and reporting your results in text, tables, or plots, targeting a generic reader who may use different software or may not know R at all. If you need a reminder on how to hide code, format tables, etc., make sure to review the <a href="https://uoepsy.github.io//rmd-bootcamp/">rmd bootcamp</a>.</p>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Important - Write-Up Examples &amp; Plagiarism
</div>
</div>
<div class="callout-body-container callout-body">
<p>The example write-up sections included below are not <strong>perfect</strong> - they instead should give you a good example of what information you should include within each section, and how to structure this. For example, some information is missing (e.g., interpretation of descriptive statistics, what type of interaction is present), some information could be presented more clearly (e.g., variable names in tables, table/figure titles/captions, and rationales for choices), and writing could be more concise in places (e.g., discussion section is quite long).</p>
<p>Further, <strong>you must not copy any of the write-up included below for future reports</strong> - if you do, you will be committing plagiarism, and this type of academic misconduct is taken very seriously by the University. You can find out more <a href="https://www.ed.ac.uk/academic-services/students/conduct/academic-misconduct">here</a>.</p>
</div>
</div>
<section id="study-overview" class="level2"><h2 class="anchored" data-anchor-id="study-overview">Study Overview</h2>
<blockquote class="blockquote">
<p><strong>Research Question</strong></p>
<p>Does the probability of having senility symptoms change as a function of the WAIS score?</p>
</blockquote>
<div class="optional-begin">
<span id="opt-start-1" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-1', 'opt-start-1')"> <span class="olab">Senility: Data Codebook</span></span>
</div>
<div id="opt-body-1" class="optional-body" style="display: none;">
<p><strong>Description</strong></p>
<p>A sample of elderly people was given a psychiatric examination to determine if symptoms of senility were present. Other measurements taken at the same time included the score on a subset of the Wechsler Adult Intelligent Scale (WAIS). The data represent symptoms of senility (senility = 1 if symptoms are present and senility= 0 otherwise) and WAIS scores (wais) for n=54 people.</p>
<p>The data in <code>SenilityWAIS.csv</code> contain seven attributes collected from a sample of <span class="math inline">\(n=54\)</span> participants:</p>
<ul>
<li>
<code>wais</code>: Score on WAIS</li>
<li>
<code>senility</code>: Whether symptoms of senility were (<code>senility = 1</code>) or were not (<code>senility = 0</code>) present</li>
</ul>
<p><strong>Preview</strong></p>
<p>The first six rows of the data are:</p>
<div class="cell">
<div class="cell-output-display">

<div id="pqxkbfqhgj" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>html {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
}

:where(#pqxkbfqhgj) .gt_table {
  display: table;
  border-collapse: collapse;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

:where(#pqxkbfqhgj) .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

:where(#pqxkbfqhgj) .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 0;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

:where(#pqxkbfqhgj) .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

:where(#pqxkbfqhgj) .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

:where(#pqxkbfqhgj) .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

:where(#pqxkbfqhgj) .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

:where(#pqxkbfqhgj) .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

:where(#pqxkbfqhgj) .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

:where(#pqxkbfqhgj) .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

:where(#pqxkbfqhgj) .gt_from_md > :first-child {
  margin-top: 0;
}

:where(#pqxkbfqhgj) .gt_from_md > :last-child {
  margin-bottom: 0;
}

:where(#pqxkbfqhgj) .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

:where(#pqxkbfqhgj) .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

:where(#pqxkbfqhgj) .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

:where(#pqxkbfqhgj) .gt_row_group_first td {
  border-top-width: 2px;
}

:where(#pqxkbfqhgj) .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

:where(#pqxkbfqhgj) .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_first_summary_row.thick {
  border-top-width: 2px;
}

:where(#pqxkbfqhgj) .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

:where(#pqxkbfqhgj) .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

:where(#pqxkbfqhgj) .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-left: 4px;
  padding-right: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

:where(#pqxkbfqhgj) .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

:where(#pqxkbfqhgj) .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

:where(#pqxkbfqhgj) .gt_left {
  text-align: left;
}

:where(#pqxkbfqhgj) .gt_center {
  text-align: center;
}

:where(#pqxkbfqhgj) .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

:where(#pqxkbfqhgj) .gt_font_normal {
  font-weight: normal;
}

:where(#pqxkbfqhgj) .gt_font_bold {
  font-weight: bold;
}

:where(#pqxkbfqhgj) .gt_font_italic {
  font-style: italic;
}

:where(#pqxkbfqhgj) .gt_super {
  font-size: 65%;
}

:where(#pqxkbfqhgj) .gt_footnote_marks {
  font-style: italic;
  font-weight: normal;
  font-size: 75%;
  vertical-align: 0.4em;
}

:where(#pqxkbfqhgj) .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

:where(#pqxkbfqhgj) .gt_indent_1 {
  text-indent: 5px;
}

:where(#pqxkbfqhgj) .gt_indent_2 {
  text-indent: 10px;
}

:where(#pqxkbfqhgj) .gt_indent_3 {
  text-indent: 15px;
}

:where(#pqxkbfqhgj) .gt_indent_4 {
  text-indent: 20px;
}

:where(#pqxkbfqhgj) .gt_indent_5 {
  text-indent: 25px;
}
</style>
<table class="gt_table">
<thead class="gt_col_headings"><tr>
<th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" scope="col" id="wais">wais</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" scope="col" id="senility">senility</th>
    </tr></thead>
<tbody class="gt_table_body">
<tr>
<td headers="wais" class="gt_row gt_right">9</td>
<td headers="senility" class="gt_row gt_right">1</td>
</tr>
<tr>
<td headers="wais" class="gt_row gt_right">13</td>
<td headers="senility" class="gt_row gt_right">1</td>
</tr>
<tr>
<td headers="wais" class="gt_row gt_right">6</td>
<td headers="senility" class="gt_row gt_right">1</td>
</tr>
<tr>
<td headers="wais" class="gt_row gt_right">8</td>
<td headers="senility" class="gt_row gt_right">1</td>
</tr>
<tr>
<td headers="wais" class="gt_row gt_right">10</td>
<td headers="senility" class="gt_row gt_right">1</td>
</tr>
<tr>
<td headers="wais" class="gt_row gt_right">4</td>
<td headers="senility" class="gt_row gt_right">1</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<p class="optional-end">
</p>
<section id="setup" class="level3"><h3 class="anchored" data-anchor-id="setup">Setup</h3>
<div class="question-begin">
Setup
</div>
<div class="question-body">
<ol type="1">
<li>Create a new RMarkdown file</li>
<li>Load the required package(s)</li>
<li>Read the SenilityWAIS dataset into R, assigning it to an object named <code>sen</code>
</li>
</ol>
</div>
<p class="question-end">
</p>
<div class="solution-begin">
<span id="sol-start-2" class="fa-solid fa-circle-arrow-right solution-icon clickable" onclick="toggle_visibility('sol-body-2', 'sol-start-2')"> Solution </span>
</div>
<div id="sol-body-2" class="solution-body" style="display: none;">
<div class="cell">
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org">tidyverse</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://personality-project.org/r/psych/">psych</a></span><span class="op">)</span> </span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="http://haozhu233.github.io/kableExtra/">kableExtra</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://strengejacke.github.io/sjPlot/">sjPlot</a></span><span class="op">)</span></span>
<span></span>
<span><span class="co">#read in data</span></span>
<span><span class="va">sen</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://readr.tidyverse.org/reference/read_delim.html">read_csv</a></span><span class="op">(</span><span class="st">"https://uoepsy.github.io/data/SenilityWAIS.csv"</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<p class="solution-end">
</p>
</section><section id="analysis-code" class="level3"><h3 class="anchored" data-anchor-id="analysis-code">Analysis Code</h3>
<p>Try to answer the research question above without referring to the provided analysis code below, and then check how your script matches up - is there anything you missed or done differently? If so, discuss the differences with a tutor - there are lots of ways to code to the same solution!</p>
<div class="optional-begin">
<span id="opt-start-3" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-3', 'opt-start-3')"> <span class="olab">Provided Analysis Code</span></span>
</div>
<div id="opt-body-3" class="optional-body" style="display: none;">
<div class="cell">
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">######Step 1 is always to read in the data, then to explore, check, describe, and visualise it.</span></span>
<span></span>
<span><span class="co">#check coding of variables - are they coded as they should be?</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/str.html">str</a></span><span class="op">(</span><span class="va">sen</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>spc_tbl_ [54 × 2] (S3: spec_tbl_df/tbl_df/tbl/data.frame)
 $ wais    : num [1:54] 9 13 6 8 10 4 14 8 11 7 ...
 $ senility: num [1:54] 1 1 1 1 1 1 1 1 1 1 ...
 - attr(*, "spec")=
  .. cols(
  ..   wais = col_double(),
  ..   senility = col_double()
  .. )
 - attr(*, "problems")=&lt;externalptr&gt; </code></pre>
</div>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">sen</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 6 × 2
   wais senility
  &lt;dbl&gt;    &lt;dbl&gt;
1     9        1
2    13        1
3     6        1
4     8        1
5    10        1
6     4        1</code></pre>
</div>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">#both variables currently coded as should be - no changes needed. </span></span>
<span></span>
<span><span class="co">#create descriptives table</span></span>
<span><span class="va">descript</span> <span class="op">&lt;-</span> <span class="va">sen</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html">%&gt;%</a></span> </span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html">group_by</a></span><span class="op">(</span><span class="va">senility</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html">%&gt;%</a></span></span>
<span>   <span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html">summarise</a></span><span class="op">(</span></span>
<span>       Mean_WAIS <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/mean.html">mean</a></span><span class="op">(</span><span class="va">wais</span><span class="op">)</span>,</span>
<span>       SD_WAIS <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/sd.html">sd</a></span><span class="op">(</span><span class="va">wais</span><span class="op">)</span>,</span>
<span>       Min_WAIS <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Extremes.html">min</a></span><span class="op">(</span><span class="va">wais</span><span class="op">)</span>,</span>
<span>       Max_WAIS <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Extremes.html">max</a></span><span class="op">(</span><span class="va">wais</span><span class="op">)</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span><span class="op">(</span>caption <span class="op">=</span> <span class="st">"Descriptive Statistics"</span>, digits <span class="op">=</span> <span class="fl">2</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://rdrr.io/pkg/kableExtra/man/kable_styling.html">kable_styling</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">descript</span></span></code></pre></div>
<div class="cell-output-display">

<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>Descriptive Statistics</caption>
 <thead><tr>
<th style="text-align:right;"> senility </th>
   <th style="text-align:right;"> Mean_WAIS </th>
   <th style="text-align:right;"> SD_WAIS </th>
   <th style="text-align:right;"> Min_WAIS </th>
   <th style="text-align:right;"> Max_WAIS </th>
  </tr></thead>
<tbody>
<tr>
<td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 12.50 </td>
   <td style="text-align:right;"> 3.46 </td>
   <td style="text-align:right;"> 4 </td>
   <td style="text-align:right;"> 20 </td>
  </tr>
<tr>
<td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 8.93 </td>
   <td style="text-align:right;"> 3.17 </td>
   <td style="text-align:right;"> 4 </td>
   <td style="text-align:right;"> 14 </td>
  </tr>
</tbody>
</table>
</div>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">#bar plot - senility presence</span></span>
<span><span class="va">sen_plt1</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span>data <span class="op">=</span> <span class="va">sen</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="fu"><a href="https://forcats.tidyverse.org/reference/as_factor.html">as_factor</a></span><span class="op">(</span><span class="va">senility</span><span class="op">)</span>, fill <span class="op">=</span> <span class="fu"><a href="https://forcats.tidyverse.org/reference/as_factor.html">as_factor</a></span><span class="op">(</span><span class="va">senility</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span> </span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_bar.html">geom_bar</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> </span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"Senility Symptoms Present (0 = No, 1 = Yes)"</span>, fill <span class="op">=</span> <span class="st">"Senility Symptoms Present (0 = No, 1 = Yes)"</span>, y <span class="op">=</span> <span class="st">"Frequency"</span><span class="op">)</span></span>
<span><span class="va">sen_plt1</span></span></code></pre></div>
<div class="cell-output-display">
<p><img src="2_09_writeup_recap_files/figure-html/unnamed-chunk-3-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">#density plot - senility &amp; WAIS</span></span>
<span><span class="va">sen_plt2</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html">ggplot</a></span><span class="op">(</span>data <span class="op">=</span> <span class="va">sen</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">wais</span>, fill <span class="op">=</span> <span class="fu"><a href="https://forcats.tidyverse.org/reference/as_factor.html">as_factor</a></span><span class="op">(</span><span class="va">senility</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span> </span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_density.html">geom_density</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> </span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html">labs</a></span><span class="op">(</span>x <span class="op">=</span> <span class="st">"WAIS Score"</span>, fill <span class="op">=</span> <span class="st">"Senility Symptoms Present (0 = No, 1 = Yes)"</span><span class="op">)</span></span>
<span><span class="va">sen_plt2</span></span></code></pre></div>
<div class="cell-output-display">
<p><img src="2_09_writeup_recap_files/figure-html/unnamed-chunk-3-2.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">######Step 2 is to run your model(s) of interest to answer your research question, and make sure that the data meet the assumptions of your chosen test</span></span>
<span></span>
<span><span class="co">#build model &amp; examine output</span></span>
<span><span class="va">sen_mdl1</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/glm.html">glm</a></span><span class="op">(</span><span class="va">senility</span> <span class="op">~</span> <span class="va">wais</span>, family <span class="op">=</span> <span class="st">"binomial"</span>, data <span class="op">=</span> <span class="va">sen</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html">summary</a></span><span class="op">(</span><span class="va">sen_mdl1</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
glm(formula = senility ~ wais, family = "binomial", data = sen)

Deviance Residuals: 
    Min       1Q   Median       3Q      Max  
-1.6702  -0.7402  -0.4749   0.5200   2.1157  

Coefficients:
            Estimate Std. Error z value Pr(&gt;|z|)   
(Intercept)   2.4040     1.1918   2.017  0.04369 * 
wais         -0.3235     0.1140  -2.838  0.00453 **
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

(Dispersion parameter for binomial family taken to be 1)

    Null deviance: 61.806  on 53  degrees of freedom
Residual deviance: 51.017  on 52  degrees of freedom
AIC: 55.017

Number of Fisher Scoring iterations: 5</code></pre>
</div>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/Log.html">exp</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/stats/coef.html">coefficients</a></span><span class="op">(</span><span class="va">sen_mdl1</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>(Intercept)        wais 
   11.06784     0.72359 </code></pre>
</div>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">#check assumptions</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">rstandard</a></span><span class="op">(</span><span class="va">sen_mdl1</span>, type <span class="op">=</span> <span class="st">"deviance"</span><span class="op">)</span>, ylab <span class="op">=</span> <span class="st">"Standardised Deviance Residuals"</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output-display">
<p><img src="2_09_writeup_recap_files/figure-html/unnamed-chunk-3-3.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/stats/influence.measures.html">cooks.distance</a></span><span class="op">(</span><span class="va">sen_mdl1</span><span class="op">)</span>, ylab <span class="op">=</span> <span class="st">"Cook's Distance"</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output-display">
<p><img src="2_09_writeup_recap_files/figure-html/unnamed-chunk-3-4.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">#compare to null - conduct model comparison</span></span>
<span><span class="co">#fit null</span></span>
<span><span class="va">sen_mdl0</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/stats/glm.html">glm</a></span><span class="op">(</span><span class="va">senility</span> <span class="op">~</span> <span class="fl">1</span>, family <span class="op">=</span> <span class="st">"binomial"</span>, data <span class="op">=</span> <span class="va">sen</span><span class="op">)</span></span>
<span><span class="co">#compare models - models are nested</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/stats/anova.html">anova</a></span><span class="op">(</span><span class="va">sen_mdl0</span>, <span class="va">sen_mdl1</span>, test <span class="op">=</span> <span class="st">"Chisq"</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Analysis of Deviance Table

Model 1: senility ~ 1
Model 2: senility ~ wais
  Resid. Df Resid. Dev Df Deviance Pr(&gt;Chi)   
1        53     61.806                        
2        52     51.017  1   10.789 0.001021 **
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/stats/AIC.html">AIC</a></span><span class="op">(</span><span class="va">sen_mdl0</span>, <span class="va">sen_mdl1</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>         df      AIC
sen_mdl0  1 63.80632
sen_mdl1  2 55.01738</code></pre>
</div>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/stats/AIC.html">BIC</a></span><span class="op">(</span><span class="va">sen_mdl0</span>, <span class="va">sen_mdl1</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>         df      BIC
sen_mdl0  1 65.79530
sen_mdl1  2 58.99535</code></pre>
</div>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">#plot model</span></span>
<span><span class="va">plt_mdl1</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://strengejacke.github.io/sjPlot/reference/plot_model.html">plot_model</a></span><span class="op">(</span><span class="va">sen_mdl1</span>, type <span class="op">=</span> <span class="st">"eff"</span><span class="op">)</span> </span>
<span><span class="va">plt_mdl1</span></span></code></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>$wais</code></pre>
</div>
<div class="cell-output-display">
<p><img src="2_09_writeup_recap_files/figure-html/unnamed-chunk-3-5.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="co">#results in formatted table</span></span>
<span><span class="fu"><a href="https://strengejacke.github.io/sjPlot/reference/tab_model.html">tab_model</a></span><span class="op">(</span><span class="va">sen_mdl1</span><span class="op">)</span></span></code></pre></div>
<div class="cell-output-display">

<table style="border-collapse:collapse; border:none;">
<tbody><tr>
<th style="border-top: double; text-align:center; font-style:normal; font-weight:bold; padding:0.2cm;  text-align:left; ">&nbsp;</th>
<th colspan="3" style="border-top: double; text-align:center; font-style:normal; font-weight:bold; padding:0.2cm; ">senility</th>
</tr>
<tr>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  text-align:left; ">Predictors</td>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  ">Odds Ratios</td>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  ">CI</td>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  ">p</td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">(Intercept)</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">11.07</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">1.23&nbsp;–&nbsp;145.24</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  "><strong>0.044</strong></td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">wais</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">0.72</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">0.56&nbsp;–&nbsp;0.89</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  "><strong>0.005</strong></td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; padding-top:0.1cm; padding-bottom:0.1cm; border-top:1px solid;">Observations</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; padding-top:0.1cm; padding-bottom:0.1cm; text-align:left; border-top:1px solid;" colspan="3">54</td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; padding-top:0.1cm; padding-bottom:0.1cm;">R<sup>2</sup> Tjur</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; padding-top:0.1cm; padding-bottom:0.1cm; text-align:left;" colspan="3">0.198</td>
</tr>
</tbody></table>
</div>
</div>
</div>
<p class="optional-end">
</p>
</section><section id="the-3-act-structure-analysis-strategy-results-discussion" class="level3"><h3 class="anchored" data-anchor-id="the-3-act-structure-analysis-strategy-results-discussion">The 3-Act Structure: Analysis Strategy, Results, &amp; Discussion</h3>
<p>Recall that we need to present our report in three clear sections - think of your sections like the 3 key parts of a play or story - we need to (1) provide some background and scene setting for the reader, (2) present our results in the context of the research question, and (3) present a resolution to our story - relate our findings back to the question we were asked and provide our answer.</p>
<p>If you need a reminder of what to include within each section, refer to <a href="https://uoepsy.github.io/dapr2/2223/labs/1_11_writeup_recap.html">Semester 1 Lab 11</a>, and read through the ‘what to include’ sections for Analysis Strategy, Results, and Discussion.</p>
<section id="act-i-analysis-strategy" class="level4"><h4 class="anchored" data-anchor-id="act-i-analysis-strategy">Act I: Analysis Strategy</h4>
<div class="question-begin">
Question 1
</div>
<div class="question-body">
<p>Attempt to draft a discussion section based on the above research question and analysis provided.</p>
</div>
<p class="question-end">
</p>
<div class="optional-begin">
<span id="opt-start-4" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-4', 'opt-start-4')"> <span class="olab">Example Write-Up of Analysis Strategy Section</span></span>
</div>
<div id="opt-body-4" class="optional-body" style="display: none;">
<p>The <code>SenilityWAIS</code> dataset contained information on 54 participants who took part in a study concerning the presence of senility symptoms (scored dichotomously as present (1) or not present (0). The sample of older adults also completed the Wechsler Adult Intelligent Scale (WAIS), and scores were available for a subset of these items.</p>
<p>To investigate whether the probability of having senility symptoms change as a function of WAIS score, a binary logistic regression model was used. Effects were considered statistically significant at <span class="math inline">\(\alpha = 0.05\)</span>. The following model specification was used:</p>
<p><span class="math display">\[
\begin{aligned}
\text{Senility} &amp;= \beta_0 \\
&amp;+ \beta_1 A_\text{AudioVideo} + \beta_2 T_\text{Trained} \\
&amp;+ \beta_3 (A_\text{AudioVideo} * T_\text{Trained}) \\
&amp;+ \epsilon
\end{aligned}
\]</span></p>
<p>To address the research question of whether the probability of having senility symptoms change as a function of WAIS score, this formally corresponded to testing whether the WAIS coefficient was equal to zero:</p>
<p><span class="math display">\[
H_0: \beta_1 = 0
\]</span></p>
<p><span class="math display">\[
H_1: \beta_1 \neq 0
\]</span></p>
<p>To assess model fit, we visually assessed the standardized deviance residuals and Cook’s Distance. We expected the former to identify outliers (or extreme values), and we expected residuals to fall within the range of -2 to 2. We used the latter to check for influential observations, and visually assessed if any of our 54 observations had a Cook’s distance &gt; than 0.5 (moderately influential) or &gt; 1 (highly influential).</p>
</div>
<p class="optional-end">
</p>
</section><section id="act-ii-results" class="level4"><h4 class="anchored" data-anchor-id="act-ii-results">Act II: Results</h4>
<div class="question-begin">
Question 2
</div>
<div class="question-body">
<p>Attempt to draft a results section based on your detailed analysis strategy and the analysis provided.</p>
</div>
<p class="question-end">
</p>
<div class="optional-begin">
<span id="opt-start-5" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-5', 'opt-start-5')"> <span class="olab">Example Write-Up of Results Section</span></span>
</div>
<div id="opt-body-5" class="optional-body" style="display: none;">
<p>Descriptive statistics are displayed in <a href="#tbl-descript">Table&nbsp;1</a>.</p>
<div class="cell" data-caption="Descriptives Table">
<div class="cell-output-display">
<div id="tbl-descript" class="anchored">

<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>Table&nbsp;1:  Descriptive Statistics </caption>
 <thead><tr>
<th style="text-align:right;"> senility </th>
   <th style="text-align:right;"> Mean_WAIS </th>
   <th style="text-align:right;"> SD_WAIS </th>
   <th style="text-align:right;"> Min_WAIS </th>
   <th style="text-align:right;"> Max_WAIS </th>
  </tr></thead>
<tbody>
<tr>
<td style="text-align:right;"> 0 </td>
   <td style="text-align:right;"> 12.50 </td>
   <td style="text-align:right;"> 3.46 </td>
   <td style="text-align:right;"> 4 </td>
   <td style="text-align:right;"> 20 </td>
  </tr>
<tr>
<td style="text-align:right;"> 1 </td>
   <td style="text-align:right;"> 8.93 </td>
   <td style="text-align:right;"> 3.17 </td>
   <td style="text-align:right;"> 4 </td>
   <td style="text-align:right;"> 14 </td>
  </tr>
</tbody>
</table>
</div>
</div>
</div>
<p>It appeared that there those without senility symptoms present had higher WAIS scores than those with symptoms (see <a href="#fig-senwais-plot">Figure&nbsp;1</a>).</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-senwais-plot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="2_09_writeup_recap_files/figure-html/fig-senwais-plot-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;1: Association between Senility Symotom Presence and WAIS</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>A binary logistic regression model was fitted to determine whether the probability of having senility symptoms change as a function of WAIS score.</p>
<p>Our model did not raise any concerns regarding fit. Though there appeared to be a few residuals with a value slightly larger than 2 in absolute value (see left-hand plot in <a href="#fig-assumpt">Figure&nbsp;2</a>), they were not influential points (see right-hand plot in <a href="#fig-assumpt">Figure&nbsp;2</a>), since none of our observations had a Cook’s distance value &gt; 0.5.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-assumpt" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="2_09_writeup_recap_files/figure-html/fig-assumpt-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;2: Model Fit Plots</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>WAIS scores were a significant predictor of whether or not individuals experienced senility symptoms (see <a href="#tbl-res">Table&nbsp;2</a>).</p>
<div class="cell" data-caption="Model Results">
<div class="cell-output-display">
<div id="tbl-res" class="anchored">

<table style="border-collapse:collapse; border:none;">
<caption style="font-weight: bold; text-align:left;">Table&nbsp;2:  Regression Table for Senility Model </caption>
<tbody><tr>
<th style="border-top: double; text-align:center; font-style:normal; font-weight:bold; padding:0.2cm;  text-align:left; ">&nbsp;</th>
<th colspan="3" style="border-top: double; text-align:center; font-style:normal; font-weight:bold; padding:0.2cm; ">Senility Symptoms</th>
</tr>
<tr>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  text-align:left; ">Predictors</td>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  ">Odds Ratios</td>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  ">CI</td>
<td style=" text-align:center; border-bottom:1px solid; font-style:italic; font-weight:normal;  ">p</td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">(Intercept)</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">11.07</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">1.23&nbsp;–&nbsp;145.24</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  "><strong>0.044</strong></td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; ">WAIS</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">0.72</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  ">0.56&nbsp;–&nbsp;0.89</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:center;  "><strong>0.005</strong></td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; padding-top:0.1cm; padding-bottom:0.1cm; border-top:1px solid;">Observations</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; padding-top:0.1cm; padding-bottom:0.1cm; text-align:left; border-top:1px solid;" colspan="3">54</td>
</tr>
<tr>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; text-align:left; padding-top:0.1cm; padding-bottom:0.1cm;">R<sup>2</sup> Tjur</td>
<td style=" padding:0.2cm; text-align:left; vertical-align:top; padding-top:0.1cm; padding-bottom:0.1cm; text-align:left;" colspan="3">0.198</td>
</tr>
</tbody></table>
</div>
</div>
</div>
<div class="cell" data-caption="ANOVA Output">
<div class="cell-output cell-output-stdout">
<pre><code>$wais</code></pre>
</div>
<div class="cell-output-display">
<div id="fig-sen-out" class="quarto-figure quarto-figure-center anchored">
<figure class="figure"><p><img src="2_09_writeup_recap_files/figure-html/fig-sen-out-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;3: Model Results</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Results suggested that for every additional point scored on the WAIS, the odds of having senility symptoms decreased by a factor of 0.72 (see <a href="#fig-sen-out">Figure&nbsp;3</a>).</p>
</div>
<p class="optional-end">
</p>
</section><section id="act-iii-discussion" class="level4"><h4 class="anchored" data-anchor-id="act-iii-discussion">Act III: Discussion</h4>
<div class="question-begin">
Question 3
</div>
<div class="question-body">
<p>Attempt to draft a discussion section based on your results and the analysis provided.</p>
</div>
<p class="question-end">
</p>
<div class="optional-begin">
<span id="opt-start-6" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-6', 'opt-start-6')"> <span class="olab">Example Write-Up of Discussion Section</span></span>
</div>
<div id="opt-body-6" class="optional-body" style="display: none;">
<p>The findings indicated that, whether or not participants had symptoms of senility was influenced by WAIS score, where the presence of senility symptoms varied as a function of WAIS score - those with senility symptoms had lower WAIS scores (Mean = 8.93, SD = 3.17) than those who did not (Mean = 12.50, SD = 3.46). Our results led us to reject the null hypothesis. The direction of this association was clear - lower WAIS scores increased the chances of senility symptoms.</p>
</div>
<p class="optional-end">
</p>
</section></section></section></section><section id="section-b-weeks-1-5-recap" class="level1"><h1>Section B: Weeks 1-5 Recap</h1>
<p>In the second part of the lab, there is no new content - the purpose of the recap section is for you to revisit and revise the concepts you have learned over the last 4/5 weeks.</p>
<div class="red">
<p>Before you expand each of the boxes below, think about how comfortable you feel with each concept.</p>
</div>
<div class="optional-begin">
<span id="opt-start-7" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-7', 'opt-start-7')"> <span class="olab">Probability, Odds, Log-Odds</span></span>
</div>
<div id="opt-body-7" class="optional-body" style="display: none;">
<ul>
<li><p>The <strong>probability</strong> <span class="math inline">\(p\)</span> ranges from 0 to 1.</p></li>
<li><p>The <strong>odds</strong> <span class="math inline">\(\frac{p}{1-p}\)</span> ranges from 0 to <span class="math inline">\(\infty\)</span>.</p></li>
<li><p>The <strong>log-odds</strong> <span class="math inline">\(\log \left( \frac{p}{1-p} \right)\)</span> ranges from <span class="math inline">\(-\infty\)</span> to <span class="math inline">\(\infty\)</span>.</p></li>
</ul>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><img src="images/glm/plo.png" class="img-fluid figure-img" style="width:95.0%"></p>
<p></p><figcaption class="figure-caption">Probability, Odds and Log-odds</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In order to understand the connections among these concepts, lets work with an example where the <em>probability</em> of an event occurring is 0.2:</p>
<ul>
<li>Odds of event occurring:</li>
</ul>
<p><span class="math display">\[
\text{odds} = (\frac{0.2}{0.8}) = 0.25
\]</span></p>
<ul>
<li>Log-odds of the event occurring:</li>
</ul>
<p><span class="math display">\[
log(\frac{0.2}{0.8}) = -1.3863 \\ \ \\
\text{OR} \\ \ \\
log(0.25) = -1.3863
\]</span></p>
<ul>
<li>Probability can be reconstructed as:</li>
</ul>
<p><span class="math display">\[
(\frac{odds}{1+odds}) = (\frac{0.25}{1+0.25}) = 0.2 \\ \ \\
\text{OR} \\ \ \\
(\frac{exp(log(odds))}{1+exp(log(odds))}) = (\frac{exp(-1.3863)}{1+exp(-1.3683)}) = (\frac{0.25}{1.25}) = 0.2
\]</span></p>
<div class="blue">
<p>In <strong>R</strong>:</p>
<ul>
<li>
<p>obtain the odds</p>
<p><code>odds &lt;- 0.2 / (1 - 0.2)</code></p>
</li>
<li>
<p>obtain the log-odds for a given probability</p>
<p><code>log_odds &lt;- log(0.25)</code></p>
<p><strong>OR</strong></p>
<p><code>log_odds &lt;- qlogis(0.2)</code></p>
</li>
<li>
<p>obtain the probability from the odds</p>
<p><code>prob_O &lt;- odds / (1 + odds)</code></p>
</li>
<li>
<p>obtain the probability from the log-odds</p>
<p><code>prob_LO &lt;- exp(log_odds) / (1 + exp(log_odds))</code></p>
<p><strong>OR</strong></p>
<p><code>prob_LO &lt;- plogis(log_odds)</code></p>
</li>
</ul>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>See S2 Week 3 <a href="https://uoepsy.github.io/dapr2/2223/labs/2_01_model_comps.html">lecture</a> and <a href="https://uoepsy.github.io/dapr2/2223/lectures/dapr2_11_modelcomparison#1">lectures</a> for further details, examples, and to revise these concepts further.</p>
</div>
</div>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-8" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-8', 'opt-start-8')"> <span class="olab">Generalized Linear Models</span></span>
</div>
<div id="opt-body-8" class="optional-body" style="display: none;">
<p>Generalized linear models can be fitted in R using the <code>glm</code> function, which is similar to the <code>lm</code> function for fitting linear models. However, we also need to specify the family (i.e., link) function. There are three key components to consider:</p>
<ul>
<li>Random component / probability distribution - The distribution of the response/outcome variable. Can be from any family of distributions as listed below.</li>
<li>Systematic component / linear predictor - the explanatory/predictor variable(s) (can be continuous or discrete).</li>
<li>Link function - specifies the link between a random and systematic components.</li>
</ul>
<p><strong>Formula:</strong></p>
<p><span class="math inline">\(y_i = \beta_0 + \beta_1 x_i + + \beta_2 x_i + \epsilon_i\)</span></p>
<div class="blue">
<p><strong>In R:</strong></p>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="fu">glm</span>(y <span class="sc">~</span> x1 <span class="sc">+</span> x2, <span class="at">data =</span> data, <span class="at">family =</span> <span class="sc">&lt;</span>INSERT_FAMILY<span class="sc">&gt;</span>)</span></code></pre></div>
</div>
</div>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-9" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-9', 'opt-start-9')"> <span class="olab">Family Argument</span></span>
</div>
<div id="opt-body-9" class="optional-body" style="display: none;">
<p>The family argument takes (the name of) a family function which specifies the link function and variance function (as well as a few other arguments not entirely relevant to the purpose of this course).</p>
<p>The exponential family functions available in <strong>R</strong> are:</p>
<ul>
<li>
<code>binomial</code> (link = “logit”)</li>
<li>
<code>gaussian</code> (link = “identity”)</li>
<li>
<code>poisson</code> (link = “log”)</li>
<li>
<code>Gamma</code> (link = “inverse”)</li>
<li>
<code>inverse.gaussian</code> (link = “1/mu2”)</li>
</ul>
<p>See <code><a href="https://rdrr.io/r/stats/glm.html">?glm</a></code> for other modeling options. See <code><a href="https://rdrr.io/r/stats/family.html">?family</a></code> for other allowable link functions for each family.</p>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-10" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-10', 'opt-start-10')"> <span class="olab">Binary Logistic Regression</span></span>
</div>
<div id="opt-body-10" class="optional-body" style="display: none;">
<p>When a response (y) is binary coded (e.g., 0/1; failure/success; no/yes; fail/pass; unemployed/employed) we must use logistic regression. The predictors can either be continuous or categorical.</p>
<p><span class="math display">\[
{\log(\frac{p_i}{1-p_i})} = \beta_0 + \beta_1 \ x_{i1}
\]</span></p>
<div class="blue">
<p><strong>In R:</strong></p>
<div class="cell">
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/stats/glm.html">glm</a></span><span class="op">(</span><span class="va">y</span> <span class="op">~</span> <span class="va">x1</span> <span class="op">+</span> <span class="va">x2</span>, data <span class="op">=</span> <span class="va">data</span>, family <span class="op">=</span> <span class="va">binomial</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-11" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-11', 'opt-start-11')"> <span class="olab">Interpreting Coefficients</span></span>
</div>
<div id="opt-body-11" class="optional-body" style="display: none;">
<p>To translate log-odds to odds in order to aid interpretation, you must exponentiate (i.e., by using <code><a href="https://rdrr.io/r/base/Log.html">exp()</a></code>) the coefficients from your model.</p>
<div class="blue">
<p><strong>In R:</strong></p>
<p><code>exp(coef(modelname))</code></p>
<p>We can also use R to extract predicted probabilities for us from our models.</p>
<ul>
<li>Calculate the predicted log-odds (probabilities on the logit scale): <code>predict(model, type="link")</code>
</li>
<li>Calculate the predicted probabilities: <code>predict(model, type="response")</code>
</li>
</ul>
</div>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-12" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-12', 'opt-start-12')"> <span class="olab">Drop-in-deviance test to compare nested models</span></span>
</div>
<div id="opt-body-12" class="optional-body" style="display: none;">
<p>When moving from linear regression to more advanced and flexible models, testing of goodness of fit is more often done by comparing a model of interest to a simpler one.</p>
<p>The only caveat is that the two models need to be <strong>nested</strong>, i.e.&nbsp;one model needs to be a simplification of the other, and all predictors of one model needs to be within the other.</p>
<p>We want to compare the model we previously fitted against a model where all slopes are 0, i.e.&nbsp;a baseline model:</p>
<p><span class="math display">\[
\begin{aligned}
M_1 : \qquad\log \left( \frac{p}{1 - p} \right) &amp;= \beta_0 \\
M_2 : \qquad \log \left( \frac{p}{1 - p} \right) &amp;= \beta_0 + \beta_1 x
\end{aligned}
\]</span></p>
<div class="blue">
<p><strong>In R:</strong> We do the comparison as follows:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>mdl_reduced <span class="ot">&lt;-</span> <span class="fu">glm</span>(DV <span class="sc">~</span> <span class="dv">1</span>, <span class="at">family =</span> <span class="st">"binomial"</span>, <span class="at">data =</span> dataset)</span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>mdl_main <span class="ot">&lt;-</span> <span class="fu">glm</span>(DV <span class="sc">~</span> IV1 <span class="sc">+</span> IV2 ... <span class="sc">+</span> IV4, <span class="at">family =</span> <span class="st">"binomial"</span>, <span class="at">data =</span> dataset)</span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(mdl_red, mdl_main, <span class="at">test =</span> <span class="st">'Chisq'</span>)</span></code></pre></div>
</div>
<p>In the output we can see the residual deviance of each model. Remember the deviance is the equivalent of residual sum of squares in linear regression.</p>
</div>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-13" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-13', 'opt-start-13')"> <span class="olab">Akaike and Bayesian Information Criteria</span></span>
</div>
<div id="opt-body-13" class="optional-body" style="display: none;">
<p>Deviance measures lack of fit, and it can be reduced to zero by making the model more and more complex, effectively estimating the value at each single data point. However, this involves adding more and more predictors, which makes the model more complex (and less interpretable).</p>
<p>Typically, simpler model are preferred when they still explain the data almost as well. This is why information criteria were devised, exactly to account for both the model misfit but also its complexity.</p>
<p><span class="math display">\[
\text{Information Criterion} = \text{Deviance} + \text{Penalty for model complexity}
\]</span></p>
<p>Depending on the chosen penalty, you get different criteria. Two common ones are the Akaike and Bayesian Information Criteria, AIC and BIC respectively:</p>
<p><span class="math display">\[
\begin{aligned}
\text{AIC} &amp;= \text{Deviance} + 2 p \\
\text{BIC} &amp;= \text{Deviance} + p \log(n)
\end{aligned}
\]</span></p>
<p>where <span class="math inline">\(n\)</span> is the sample size and <span class="math inline">\(p\)</span> is the number of regression coefficients in the model. <strong>Models that produce smaller values of these fitting criteria should be preferred.</strong></p>
<p>AIC and BIC differ in their degrees of penalization for number of regression coefficients, with BIC usually favouring models with fewer terms.</p>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-14" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-14', 'opt-start-14')"> <span class="olab">Exploratory vs Confirmatory Analyses</span></span>
</div>
<div id="opt-body-14" class="optional-body" style="display: none;">
<p><em>Exploratory</em> analyses are conducted when either (1) you have a hypothesis but no clear analysis plan/strategy; (2) you have lots of variables that might be associated with an outcome variable, but you’re not sure which or in what ways i.e., no clear predictions. It also provides tools for hypothesis generation - particularly via visualisation of data.</p>
<p><em>Confirmatory</em> analyses are conducted when you have a specific research question to test. You can think of this type of analysis as putting your hypotheses to trial - does your data support or fail to support your argument?</p>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-15" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-15', 'opt-start-15')"> <span class="olab">Steps in Exploratory Analyses</span></span>
</div>
<div id="opt-body-15" class="optional-body" style="display: none;">
<p>There are a number of steps involved in exploratory analyses:</p>
<ul>
<li>Step 1: Check coding of data and visualise variables of interest</li>
<li>Step 2: Compare models of interest to find if your variables of interest are good predictors of your outcome variable</li>
<li>Step 3: Compute the k-fold cross validation MSE for each of your models</li>
<li>Step 4: Identify best fitting model</li>
</ul>
</div>
<p class="optional-end">
</p>
<div class="optional-begin">
<span id="opt-start-16" class="fa fa-chevron-circle-right optional-icon clickable" onclick="toggle_visibility('opt-body-16', 'opt-start-16')"> <span class="olab">Power</span></span>
</div>
<div id="opt-body-16" class="optional-body" style="display: none;">
<p><strong>Power for regression</strong></p>
<p>In linear regression, the relevant function in <strong>R</strong> is:</p>
<div class="cell">
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r"><code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/heliosdrm/pwr">pwr</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/pwr/man/pwr.f2.test.html">pwr.f2.test</a></span><span class="op">(</span>u <span class="op">=</span> , v <span class="op">=</span> , f2 <span class="op">=</span> , sig.level <span class="op">=</span> , power <span class="op">=</span> <span class="op">)</span></span></code></pre></div>
</div>
<p>where</p>
<ul>
<li>
<code>u</code> = numerator degrees of freedom</li>
<li>
<code>v</code> = denominator degrees of freedom</li>
<li>
<code>f2</code> = effect size</li>
</ul>
<section id="section" class="level2 unnumbered tabset"><h2 class="unnumbered tabset anchored" data-anchor-id="section"></h2>
<p>The formula for the effect size <span class="math inline">\(f^2\)</span> is different depending on your goal.</p>
<section id="goal-1.-test-on-all-slopes" class="level3"><h3 class="anchored" data-anchor-id="goal-1.-test-on-all-slopes">Goal 1. Test on all slopes</h3>
<p>Here you have one model,</p>
<p><span class="math display">\[
y = \beta_0 + \beta_1 x_1 + \dots + \beta_k x_k + \epsilon
\]</span></p>
<p>and you wish to find the minimum sample size required to answer the following test of hypothesis with a given power:</p>
<p><span class="math display">\[
\begin{aligned}
H_0 &amp;: \beta_1 = \beta_2 = \dots = \beta_k = 0 \\
H_1 &amp;: \text{At least one } \beta_i \neq 0
\end{aligned}
\]</span></p>
<p>The appropriate formula for the effect size is:</p>
<p><span class="math display">\[
f^2 = \frac{R^2}{1 - R^2}
\]</span></p>
<p>And the numerator degrees of freedom are <span class="math inline">\(\texttt u = k\)</span>, the number of predictors in the model.</p>
<p>The denominator degrees of freedom returned by the function will give you:</p>
<p><span class="math display">\[
\texttt v = n - (k + 1) = n - k - 1
\]</span></p>
<p>From which you can infer the sample size as</p>
<p><span class="math display">\[
n = \texttt v + k + 1
\]</span></p>
</section><section id="goal-2.-test-on-a-subset-of-slopes" class="level3"><h3 class="anchored" data-anchor-id="goal-2.-test-on-a-subset-of-slopes">Goal 2. Test on a subset of slopes</h3>
<p>In this case you would have a smaller model <span class="math inline">\(m\)</span> with <span class="math inline">\(k\)</span> predictors and a larger model <span class="math inline">\(M\)</span> with <span class="math inline">\(K\)</span> predictors:</p>
<p><span class="math display">\[
\begin{aligned}
m &amp;: \quad y = \beta_0 + \beta_1 x_1 + \dots + \beta_{k} x_k
+ \epsilon \\
M &amp;: \quad y = \beta_0 + \beta_1 x_1 + \dots + \beta_{k} x_k
+ \underbrace{\beta_{k + 1} x_{k+1} + \dots + \beta_{K} x_K}_{\text{extra predictors}}
+ \epsilon
\end{aligned}
\]</span></p>
<p>This case is when you wish to find the minimum sample size required to answer the following test of hypothesis:</p>
<p><span class="math display">\[
\begin{aligned}
H_0 &amp;: \beta_{k+1} = \beta_{k+2} = \dots = \beta_K = 0 \\
H_1 &amp;: \text{At least one of the above } \beta \neq 0
\end{aligned}
\]</span></p>
<p>You need to use the R-squared from the larger model <span class="math inline">\(R^2_M\)</span> and the R-squared from the smaller model <span class="math inline">\(R^2_m\)</span>. The appropriate formula for the effect size is:</p>
<p><span class="math display">\[
f^2 = \frac{R^2_{M} - R^2_{m}}{1 - R^2_M}
\]</span></p>
<p>Here, the numerator degrees of freedom are the extra predictors: <span class="math inline">\(\texttt u = K - k\)</span>.</p>
<p>The denominator degrees of freedom returned by the function will give you (here you use <span class="math inline">\(K\)</span> the number of all predictors in the larger model):</p>
<p><span class="math display">\[
\texttt v = n - (K + 1) = n - K - 1
\]</span></p>
<p>From which you can infer the sample size as</p>
<p><span class="math display">\[
n = \texttt v + K + 1
\]</span></p>
</section><section id="generic-guidelines" class="level3"><h3 class="anchored" data-anchor-id="generic-guidelines">3. Generic guidelines</h3>
<p>But what can you do if you have no clue what effect size to expect in a given study? Cohen (1988) <a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> provided guidelines for what a small, medium, or large effect typically is in the behavioural sciences.</p>
<table class="table">
<thead><tr class="header">
<th style="text-align: left;">Type of test</th>
<th style="text-align: center;">Small</th>
<th style="text-align: center;">Medium</th>
<th style="text-align: center;">Large</th>
<th style="text-align: center;"></th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<span class="math inline">\(t\)</span>-test</td>
<td style="text-align: center;">0.20</td>
<td style="text-align: center;">0.50</td>
<td style="text-align: center;">0.80</td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">ANOVA</td>
<td style="text-align: center;">0.10</td>
<td style="text-align: center;">0.25</td>
<td style="text-align: center;">0.40</td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">Linear regression</td>
<td style="text-align: center;">0.02</td>
<td style="text-align: center;">0.15</td>
<td style="text-align: center;">0.35</td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
<p>For more information, please refer to Cohen J. (1992). <a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a></p>
</section></section><section id="section-1" class="level2 unnumbered"><h2 class="unnumbered anchored" data-anchor-id="section-1"></h2>
</section>
</div>
<p class="optional-end">
</p>


</section><div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>
<ol>
<li id="fn1"><p>Cohen, J. (1988) <em>Statistical Power Analysis for the Behavioral Sciences</em>, 2nd ed.&nbsp;Hillsdale, NJ: Lawrence Erlbaum.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p>Cohen J. (1992) A power primer, <em>Psychological Bulletin, 112</em>(1): 155–159. PDF available via the University of Edinburgh library at <a href="https://discovered.ed.ac.uk/permalink/f/1s15qcp/TN_cdi_proquest_journals_614317877">this link.</a><a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol></section></div></main><!-- /main --><script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer"><div class="nav-footer">
      <div class="nav-footer-center">© Copyright 2019-2022 <a href="https://www.ed.ac.uk/">The University of Edinburgh</a>. Site licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU AGPLv3</a> license.</div>
  </div>
</footer>


</body></html>